src/actors/client/ActorClient/ActorClient.ts

```diff
@@ -0,0 +1,36 @@
+import { DaprClient } from '../../..';
+import CommunicationProtocolEnum from '../../../enum/CommunicationProtocol.enum';
+import GRPCClient from '../../../implementation/Client/GRPCClient/GRPCClient';
+import HTTPClient from '../../../implementation/Client/HTTPClient/HTTPClient';
+import IClient from '../../../interfaces/Client/IClient';
+import IClientActor from '../../../interfaces/Client/IClientActor';
+import ActorClientGRPC from './ActorClientGRPC';
+import ActorClientHTTP from './ActorClientHTTP';
+
+export default class ActorClient {
+  readonly daprClient: DaprClient;
+  readonly actor: IClientActor;
+
+  constructor(daprClient: DaprClient) {
```

Comment:
What is the advantage of passing this object here just to get it later? Is there a way to get the host and port directly and avoid requiring the actual general purpose Dapr client?

Reply:
Well, the DaprClient could be transformed into a Singleton if that makes more sense? In the end there should be only one active throughout the code. Would that make sense?

Reply:
Just let the implementation of the ActorClient to solve how to keep a reference to DaprClient. Not every actor implementation should need this instance.

Reply:
Ah now I see it, apologies, was sleeping I think when writing this 😅😂